.github/workflows/spread-tests.yaml

```diff
@@ -254,12 +259,16 @@ jobs:
       run: |
         if [ -e spread.log ]; then
             echo "Running spread log analyzer"
+            fundamental="#notfundamental"
+            if [ "${{ inputs.is-fundamental }}" == 'true' ]; then
+              fundamental="#fundamental"
+            fi
             ACTIONS_URL="${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}/attempts/${{ github.run_attempt }}"
             ./tests/lib/external/snapd-testing-tools/utils/log-parser spread.log --output spread-results.json --cut 1 >/dev/null
             while IFS= read -r line; do
                 if [ ! -z "$line" ]; then
                     echo "Adding failed test line to filtered log"
-                    echo "GRAFANA FAILED: $line $ACTIONS_URL" | tee -a "$FILTERED_LOG_FILE"
+                    echo "GRAFANA FAILED: $line $ACTIONS_URL" $fundamental | tee -a "$FILTERED_LOG_FILE"
```

Comment:
I think we also need to add _fund_ and _notfund_ to the filtered log name, so then we can filter by filename in grafana (to avoid maintain that lists).